tests/unit_tests.py

```diff
@@ -99,14 +100,15 @@ def create_Partner(create_all_datasets):
 
 
 @pytest.fixture(scope="class",
-                params=((Mnist, ["basic", "random"], ['not-corrupted'] * 3),
-                        (Mnist, ["basic", "random"],
-                         ['permutation', Redundancy(0.2), Duplication(duplicated_partner_id=0)]),
-                        (Mnist, ["advanced", [[4, "specific"], [6, "shared"], [4, "shared"]]], ['not-corrupted'] * 3),
-                        (Cifar10, ["basic", "random"], ['not-corrupted'] * 3),
-                        (
-                                Cifar10, ["advanced", [[4, "specific"], [6, "shared"], [4, "shared"]]],
-                                ['not-corrupted'] * 3)),
+                params=((Mnist, "random", ['not-corrupted'] * 3),
+                        (Mnist, "random", ['permutation', Redundancy(0.2), Duplication(duplicated_partner_id=0)]),
+                        (Mnist,
+                         AdvancedSplitter([0.3, 0.5, 0.2], [[4, "specific"], [6, "shared"], [4, "shared"]]),
+                         ['not-corrupted'] * 3),
+                        (Cifar10, "random", ['not-corrupted'] * 3),
+                        (Cifar10,
+                         AdvancedSplitter([0.3, 0.5, 0.2], [[4, "specific"], [6, "shared"], [4, "shared"]]),
```

Comment:
So only `AdvancedSplitter` is tested here?

Reply:
No, random is tested too

Reply:
But indeed we should test stratified too